v2.0/resourcepacks/create_astral_translation/assets/farmersdelight/lang/zh_cn.json

```
@@ -1,5 +1,26 @@
 {
   "block.farmersdelight.beetroot_crate": "箱装玉米",
   "block.farmersdelight.wild_beetroots": "海玉米",
-  "item.farmersdelight.vegetable_noodles": "古扎拉布拉面"
+  "block.farmersdelight.rice_roll_medley_block": "寿司拼盘",
+  "block.farmersdelight.sandy_shrub": "沙生灌木",
+
+  "item.farmersdelight.vegetable_noodles": "蔬菜拉面",
```

```suggestion
  "item.farmersdelight.vegetable_noodles": "歪比巴卜面",
```
整合包将蔬菜拉面改为了Guljalabala，没查到什么特殊释义，翻译为“歪比巴卜面”。